rzup/rzup

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env bash
+
+MIN_RUST_VERSION="1.74.1" # Minimum required Rust version
```

Comment:
Is this correct? We should note this in our docs as the MSRV (and in the main README)

Reply:
After asking @SchmErik about this I found out we do not have a specified MSRV for `cargo-risczero` host installation. I tested our install msrv with a script for every rust version `nightly`-`1.1.0` and found that it fails at versions <`1.60.0` due to `cargo-binstall` support (in hindsight I should have just checked crates.io). So I recommend we change this to variable `1.60.0`, if the check is worth keeping at all. Also okay to modify it simply to check if rust is installed, and let the user find out they are very out-of-date on their own.

Reply:
I went with the simpler option, and now just check the rust installation.